js/packages/web/src/views/auctionCreate/priceAuction.tsx

```diff
@@ -16,72 +40,88 @@ export const PriceAuction = (props: {
 
       {props.attributes.category === AuctionCategory.Open && (
         <label>
-          <span>Price</span>
-          <span>
+          <h3>Price</h3>
+          <p>
             This is the fixed price that everybody will pay for your
             Participation NFT.
-          </span>
-          <Input
-            type="number"
+          </p>
+          <InputNumber<number>
             min={0}
+            decimalSeparator="."
+            className="metaplex-fullwidth"
+            step="0.01"
             autoFocus
-            placeholder="Fixed Price"
-            prefix="◎"
-            suffix="SOL"
-            onChange={info =>
+            placeholder="Fixed Price in SOL"
+            onChange={price => {
+              handleInput(price, true);
               props.setAttributes({
                 ...props.attributes,
                 // Do both, since we know this is the only item being sold.
-                participationFixedPrice: parseFloat(info.target.value),
-                priceFloor: parseFloat(info.target.value),
-              })
-            }
+                participationFixedPrice: price,
+                priceFloor: price,
+              });
+            }}
+            precision={2}
+            formatter={x => (x ? `◎ ${x}` : '')}
           />
         </label>
       )}
       {props.attributes.category !== AuctionCategory.Open && (
         <label>
           <h3>Price Floor</h3>
           <p>This is the starting bid price for your auction.</p>
-          <Input
-            type="number"
+          <InputNumber<number>
             min={0}
+            decimalSeparator="."
+            className="metaplex-fullwidth"
+            step="0.01"
             autoFocus
-            placeholder="Price"
-            prefix="◎"
-            suffix="SOL"
-            onChange={info =>
+            placeholder="Price in SOL"
+            onChange={price => {
+              handleInput(price, true);
               props.setAttributes({
                 ...props.attributes,
-                priceFloor: parseFloat(info.target.value),
-              })
-            }
+                priceFloor: price,
+              });
+            }}
+            precision={2}
+            formatter={x => (x ? `◎ ${x}` : '')}
           />
         </label>
       )}
       <label>
-        <h3>Tick Size</h3>
+        <h3 className="metaplex-margin-top-4">Tick Size</h3>
         <p>All bids must fall within this price increment.</p>
-        <Input
-          type="number"
+        <InputNumber<number>
           min={0}
+          decimalSeparator="."
+          className="metaplex-fullwidth"
+          step="0.01"
```

Comment:
Can the validation for tick size be done as a validation function attached to the input or do we not have this part of the fields wrapped in a form yet? Using a form here like we did with the nft descriptive fields is a good call.